content/module-reference/processing-modules/filmic-rgb.md

```diff
@@ -2,62 +2,73 @@
 title: filmic rgb
 id: filmic-rgb
 applicable-verison: 3.2.1
-tags: 
+tags:
 working-color-space: RGB
 view: darkroom
 masking: true
 ---
-Compress the tonal range of an image by reproducing the tone and color response of classic film. 
+Remap the tonal range of an image by reproducing the tone and color response of classic film.
+It can be used either to expand or to contract the dynamic range of the scene into the
+dynamic range of the display.
```

Comment:
Please don't include line feeds within paragraphs. If you need line wrapping please enable this in your editor. There are a few instances of this which I'll correct.